packages/react-composites/src/composites/CallComposite/components/MediaGallery.tsx

```diff
@@ -131,6 +131,8 @@ export const MediaGallery = (props: MediaGalleryProps): JSX.Element => {
         remoteVideoTileMenuOptions={remoteVideoTileMenuOptions}
         /* @conditional-compile-remove(vertical-gallery) */
         overflowGalleryPosition={overflowGalleryPosition}
+        /* @conditional-compile-remove(click-to-call) */
+        localVideoTileSize={props.isMobile ? 'matchStream' : '16:9'}
```

Comment:
why not matchStream on desktop as well?

Reply:
desktop web cams do not know when the stream has been rotated and the standard aspect ratio for most camera's is 16:9 so we want to enforce this aspect ratio with desktop mode.

Reply:
Yeah makes sense, if it was matching the stream aspect ratio I would say pick matchStream but as its following orientation then yeah keep 16:9